src/contact.rs

```diff
@@ -1691,8 +1696,7 @@ mod tests {
         assert_eq!(contact.get_name_n_addr(), "Wonderland, Alice (alice@w.de)");
 
         // check SELF
-        let contact = Contact::load_from_db(&t, DC_CONTACT_ID_SELF).await.unwrap();
-        assert_eq!(DC_CONTACT_ID_SELF, ContactId::new(1));
```

Comment:
see above, however, keeping the constant-check at one place is sufficient (but additional checks won't hurt :)